src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.VirtualAlloc.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        internal const int MEM_COMMIT = 0x1000;
+        internal const int MEM_RESERVE = 0x2000;
+        internal const int MEM_RELEASE = 0x8000;
+        internal const int MEM_FREE = 0x10000;
+        internal const int PAGE_READWRITE = 0x04;
+
+#if ENABLE_WINRT
+        [DllImport(Libraries.Kernel32, EntryPoint = "VirtualAllocFromApp")]
```

Comment:
`VirtualAllocFromApp` is not in Kernel32 - the MSDN documentation is a lie (I would file a bug, but I don't see the link - pretty sure there used to be a link).

This is exposed from api-ms-win-core-memory-l1-1-3.dll and higher only. I know our policy seems to be no apisets, but there's no way around that. Can I just add a `Memory` entry in the `Libraries` class with that value or is there a different way?

Reply:
I do not think we need to have a constant for `api-ms-win-core-memory-l1-1-3.dll`. I would just use this string in this one DllImport directly.

Reply:
I changed it only because of the MSDN documentation. The code is ProjectN only, so I assume it's ok to put back the apiset name (and comment).

Reply:
I don't think this bug was preventable.

We could prevent this class of bugs: #6878 by adding a CSPROJ that builds CoreLib the way Project N builds it to the CoreRT repo, but in general, @jkotas is almost as good as Roslyn at spotting those, so it hasn't been too critical.

Reply:
I do have a modified .csproj locally to do ENABLE_WINRT builds, but I often fail to check it before submitting a PR. It may not match exactly Project N either.